Docker/Ganache.Dockerfile

```diff
@@ -7,5 +7,5 @@ COPY ./on-chain/evm-contracts .
 
 RUN yarn install --silent && node_modules/.bin/truffle compile
 
-ENTRYPOINT ["node_modules/.bin/ganache-cli", "-h", "0.0.0.0", "-p","8545","-m","when sound uniform light fee face forum huge impact talent exhaust arrow"]
```

Comment:
Keep this please

Reply:
It should probably match the keys used elsewhere?